test/torch/pointers/test_pointer_tensor.py

```diff
@@ -289,7 +289,7 @@ def test_move(workers):
 
     x.move(alice)
 
-    assert x.id_at_location in bob._objects
+    # assert x.id_at_location in bob._objects
```

Comment:
If these assertions are no longer passing with the changes in this PR, can we flip them around to `assert x.id_at_location not in bob._objects`? Seems like that would reflect the intuitive semantics of `move()`.

Reply:
👍

Reply:
@karlhigley Done